modules/dnn/src/layers/normalize_bbox_layer.cpp

```diff
@@ -360,5 +360,67 @@ Ptr<NormalizeBBoxLayer> NormalizeBBoxLayer::create(const LayerParams &params)
     return Ptr<NormalizeBBoxLayer>(new NormalizeBBoxLayerImpl(params));
 }
 
+
+class ChannelNormLayerImpl CV_FINAL : public ChannelNormLayer
```

Comment:
What do you think if instead of adding this custom layer we can modify `.ptototxt` replacing it to series of layers that give the sample result? In example,
```
x*x -> reshape (1, 1, C, -1) -> ave pool by H -> multiply by C -> reshape back
```

Reply:
Or, which is better, to do the same in the immporter

Reply:
In this case, we need to store `outShapes `in importer.  Maybe we will have problems with nets without `.caffemodel`.

Reply:
> Maybe we will have problems with nets without .caffemodel.
Why so?

Reply:
We have already tried to calculate outShapes in caffe_importer. Some tests failed in `getMemoryShapes` method if we don't use .caffemodel for nets with weighted layers.

Reply:
I modified `channel_norm.prototxt ` to a series of layers.